scipy/stats/tests/test_multivariate.py

```diff
@@ -2523,11 +2526,11 @@ def test_process_parameters(self):
 
         message = "each element of `row` must be an integer"
         with pytest.raises(ValueError, match=message):
-            random_table([2.1, 2.1], [1, 1, 2])
+            random_table([2.1, 1.9], [1, 1, 2])
```

Comment:
This change was necessary to pass the previous check, which checks that the column sum and row sum is equal. Before there were, because the doubles were rounded down automatically in the conversion from double to int. Now that I changed the dtype, the sum is calculated exactly and I had to adjust the numbers.

Reply:
I see, and you want to make sure it gets to the "must be an integer" error.